tests/test_lookupFunction.py

```diff
@@ -72,12 +70,13 @@ def testPrerequisiteLookupFunction(self):
         """
         butler = dafButler.Butler(os.path.join(getPackageDir("ci_hsc_gen3"), "DATA", "butler.yaml"))
 
-        pipeline = pipeBase.Pipeline("Test LookupFunciton Pipeline")
+        pipeline = pipeBase.Pipeline("Test LookupFunction Pipeline")
         pipeline.addTask(LookupTestPipelineTask, "test")
 
         graphBuilder = pipeBase.GraphBuilder(butler.registry)
-        graph = graphBuilder.makeGraph(pipeline, defaultdict(lambda: ("calib/hsc", "shared/ci_hsc_outputs")),
-                                       "test_output", None)
+        collections = dafButler.CollectionSearch.fromExpression(("calib/hsc", "shared/ci_hsc_output"))
+        graph = graphBuilder.makeGraph(pipeline, collections,
+                                       None, None)
```

Comment:
In case you're curious: passing `None` as the third argument here instead of `"test_output"` is indeed an improvement, as the test shouldn't care what's passed there (and I don't know if `None` was usable here before, but it is now).